src/g_ltac_iter.mlg

```diff
@@ -76,11 +80,11 @@ struct
             Hints.Hint_db.iter (fun _ _ hl -> syms := hl :: !syms) db ;
             !syms
           with Not_found ->
-            let _ = Tacticals.tclIDTAC_MESSAGE Pp.(str "Hint database " ++ str db_name ++ str " not found!") in
+            (* let _ = Tacticals.tclIDTAC_MESSAGE Pp.(str "Hint database " ++ str db_name ++ str " not found!") in *)
```

Comment:
Perhaps the right thing to do here is simply use the feedback mechanism.

```ocaml
Feedback.msg_warning Pp.(str "Hint database " ++ str db_name ++ str " not found!") ;
```

I guess if you want to be good, you should record the location information and report it as well.